signalfx-org-metrics/metrics.yaml

```diff
@@ -1111,12 +1135,22 @@ sf.org.numLimitedMetricTimeSeriesCreateCalls:
   metric_type: counter
   title: sf.org.numLimitedMetricTimeSeriesCreateCalls
 
+sf.org.numLimitedMetricTimeSeriesCreateCallsByCategoryType:
+  brief: Number of MTS not created because your account reached a category limit (or subscription limit).
```

Comment:
Might be helpful to detail out what 'category' is here because it seems like we generally use 'category' to mean 'metric type' in the docs. @udaysagar2177 should be able to give more insight.